packages/paste-codemods/tools/.cache/mappings.json

```diff
@@ -21,7 +21,9 @@
   "ChatAttachment": "@twilio-paste/core/chat-log",
   "ChatAttachmentDescription": "@twilio-paste/core/chat-log",
   "ChatAttachmentLink": "@twilio-paste/core/chat-log",
+  "ChatBookend": "@twilio-paste/core/chat-log",
```

Comment:
question: with `ChatBookendItem` do you need to run the build again?

Reply:
yuppp, just did